qcodes/dataset/data_set.py

```diff
@@ -353,7 +369,7 @@ def sample_name(self) -> str:
         return get_sample_name_from_experiment_id(self.conn, self.exp_id)
 
     @property
-    def run_timestamp_raw(self) -> float:
+    def run_timestamp_raw(self) -> Optional[float]:
```

Comment:
Too late but ---- the same kind of change has to also be made for `def run_timestamp` method - it has to handle the None case nicely (perhaps also return None instead of formattted time string). I wonder why it wasn't caught by tests :(((

Reply:
Aiiii, good catch!

The `run_timestamp` will work for `None`; that just gives CURRENT TIME. Silent data corruption, omg everyone freak out!

I'll make a quickfix PR.